packages/ui/src/1_atoms/Input/Input.module.css

```diff
@@ -1,63 +1,39 @@
 .wrapper {
-  @apply w-full rounded-lg max-w-xs border flex flex-row justify-between items-center;
+  @apply w-full max-w-xs flex flex-row justify-between items-center font-roboto;
 }
 
 .input {
-  @apply w-full h-8 py-2 px-3 leading-none text-base font-semibold text-gray-1 bg-sov-white duration-300 hover:bg-gray-10;
-}
-.input::placeholder {
-  @apply text-gray-8;
-}
+  @apply w-full rounded px-3 leading-tight text-xs font-medium text-gray-10 bg-gray-70 hover:bg-gray-60;
 
-.input[value=""]:not(:focus),
-.input:not([value]):not(:focus) {
-  @apply bg-gray-9;
-}
+  &::placeholder {
+    @apply text-gray-30;
+  }
 
-.input:disabled {
-  @apply cursor-not-allowed bg-gray-8 text-gray-6;
-}
+  &:focus {
+    @apply outline outline-1 outline-gray-50 bg-gray-60;
+  }
 
-.input:read-only:not(:disabled) {
-  @apply bg-transparent text-sov-white;
-}
+  &:disabled {
+    @apply cursor-not-allowed bg-gray-70-ol text-gray-10/50;
 
-.input[type="number"] {
-  appearance: textfield;
-}
-.input[type="number"][step] {
-  @apply pr-7;
-}
-.input[type="number"]::-webkit-inner-spin-button,
-.input[type="number"]::-webkit-outer-spin-button {
-  -webkit-appearance: none;
-  margin: 0;
-}
-
-.stepButton {
-  @apply absolute right-1 flex items-center justify-center h-4 w-6 
-    transition-opacity duration-300 hover:opacity-50;
+    &::placeholder {
+      @apply text-gray-30/50;
+    }
+  }
 
-  padding: 3px;
-}
-.stepButton > span {
-  @apply w-0 h-0;
-}
+  &:read-only:not(:disabled) {
+    @apply bg-gray-70/50 outline outline-1 outline-gray-50 text-gray-30;
+  }
 
-.stepButton.up {
-  @apply top-0 items-end;
-}
-.stepButton.up > span {
-  border-left: 5px solid transparent;
-  border-right: 5px solid transparent;
-  border-bottom: 5px solid theme("colors.black");
+  &[type='number'] {
+    &::-webkit-inner-spin-button,
+    &::-webkit-outer-spin-button {
+      -webkit-appearance: none;
+      margin: 0;
+    }
```

Comment:
add pls after webkit styles this prop ` -moz-appearance: textfield;` to remove arrows/spinners from input in Firefox

Reply:
Nice catch Pietro, thanks :) I was wondering why it was needed before, now I know it :D